projects/jetstack/cert-manager/helm/sedfile.template

```diff
@@ -0,0 +1,3 @@
+s,quay.io,${HELM_REGISTRY},g
```

Comment:
I guess an alternative would be to copy the upstream values.yaml and make whatever changes we want there, instead of dynamically making the change with sed at build time?  Helm doesnt support multiple values.yaml does it where we could apply our specific values on top of the upstream?

I dont have a strong opinion against what you are doing, just trying to wrap my head around it a bit.

Reply:
The sed search and replace is mainly to replace registry and image tag at build time because those depend on the actual build, but I tried to leave it flexible in case we want to use that for something else.

Reply:
Yeah that makes sense, I was just wondering if helm supported multiple values files to avoid having to do this at all and let helm take care of it?

Reply:
My understanding is there are 3 values files in order of what has say:
* Users values file overrides everything
* Parent chart values file which overrides the default
* This values file which is the default

We could also always override the values as the "user", but I think for security purposes, we want it so when we install v1.1.0-1 cert manager it installs the v1.1.0-1 container images all the time. We could in theory make every add-on a nested chart, but that seems messy for what we want to do.  We won't allow the user to override the images in a helm chart using the controller, but obviously, the user can do whatever they like if they were using the chart outside of that.

Reply:
ok yeah that makes sense. Thanks for the explanation.